src/features/page_view_timing/aggregate/index.js

```diff
@@ -196,7 +196,7 @@ export class Aggregate extends AggregateBase {
   onHarvestFinished (result) {
     if (result.retry && this.timingsSent.length > 0) {
       for (var i = 0; i < this.timingsSent.length; i++) {
-        this.timings.push(this.timingsSent[i])
+        this.timings.unshift(this.timingsSent[i])
       }
```

Comment:
`this.timings` was filled in chronological order aka per call to `addTiming`, and that's the order in which it should be resent out, right?
If so, the unshift 1-by-1 actually reverses the order of the sent timings. This can do what Trace has done, simply:
````javascript
this.timings = this.timingsSent.concat(this.timings)

Reply:
Or is `this.timings` wrong too? Should it be `[3,2,1]` rather than `[1,2,3]` ?

Reply:
Good catch @cwli24 the loop actually isn't needed at all.